playbooks/ganesha-bootstrap-new-nodes.yml

```diff
@@ -51,11 +51,19 @@
     service: name=NetworkManager enabled=no
     ignore_errors: yes
 
+  - name: Start network manager service
+    service: name=NetworkManager state=started
+
+  - name: Enable network manager service
+    service: name=NetworkManager enabled=yes
+
   - name: Start network service
     service: name=network state=started
+    ignore_errors: yes
```

Comment:
Same here

Reply:
so from rhel8, network service, is not used, it is handled by Network Manager service,
unless otherwise configured.
so failure to start is expected. going forward this entire network service will be removed, and only network manager will be there